app/scripts/controllers/network-set-controller.js

```diff
@@ -48,6 +55,7 @@ ndexApp.controller('networkSetController',
         setTooltip("Copy network set share URL to clipboard");
     };
 
+    $scope.renderNetworkTable = false;
```

Comment:
Please use a consistent style in your code. If you are using networkSetController.xxx style. You should use it throughout. Please try to avoid mixing the $scope style with it.